IFC4x3/Properties/l/LastAssessmentReport_1gG02PahnF2fsfAbELSVmv/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LastAssessmentReport_1gG02PahnF2fsfAbELSVmv" Name="LastAssessmentReport" UniqueId="6a400099-92bc-4f0a-9da9-2a539571fc39" PrimaryDataType="IfcLabel" />
```

Comment:
it is some sort of reference to the last assessment report not an extensive textual entry, it could be an IfcExternalReference but i do not want to constrict it to just that.